### ISSUE TITLE
Create a test suite for each supported language

### ISSUE THREAD

**Issue by shobrook** [shobrook/rebound#6]

**Comment by shreyasbapat** [comment]
Do you want to set up the CI or just add the unit tests for each module?

**Comment by shobrook** [comment]
Just unit tests for now, but I'd like to setup CI in the future.

**Comment by maxthecabbie** [comment]
@shobrook are you just wanting to check that the `get_error_message` function pulls the correct exception/error message when given the stack trace string from an exception?

**Comment by shobrook** [comment]
Yes, I'd like to make sure this works for every type of exception. The test suite would look like a set of test files for each language supported by `rebound`.

**Comment by shankar-lucideus** [comment]
Is this resolved yet @shobrook? Else I'd like to work on this.
